v2.2/load-based-splitting.md

```diff
@@ -0,0 +1,78 @@
+
+On the other hand, some workloads with very large machines might want to increase the QPS threshold to split more conservatively.
+
+## How load-based splitting works
+
+Whenever a range exceeds the cluster's setting for `kv.range_split.load_qps_threshold`, the range becomes eligible for load-based splitting.
+
+At that point, we begin gathering per-key metrics to determine whether or not a split would benefit the cluster's performance based on the following metrics:
+
+- **Balance factor:** If we perform a split, would there be a balance of load on both sides of the split? For example, if 99% of the traffic is for a single key, splitting it apart from the others in the range will not have a substantial impact on performance. However, if 60% of the queries are for a single key, it is likely to benefit the cluster to move the key to its own range.
+
+- **Split crossing:** If we perform a split, how many queries would have to cross this new range boundary? Because involving multiple ranges in a query incurs greater overhead than a single range, splitting a range could actually degrade performance. For example, if the range is involved in many `SELECT COUNT(*)...` operations, splitting the range in two could negatively impact performance.
+
+### Determining a split point
+
+Using the per-key metrics gathered once a node exceeds the `kv.range_split.load_qps_threshold` value, we estimate a good place to perform a split by determining the total operations over the first set of keys that exceed the `kv.range_split.load_qps_threshold`.
+
+For example, if the operations on a single key exceed the `kv.range_split.load_qps_threshold` value, it's a good candidate to split the range at that point.
+
+Another example is that if the range has equal access among all keys, whose total operation exceeds `kv.range_split.load_qps_threshold`. By splitting the range at the first set of keys whose total operations exceed the threshold, we can reduce the load on the node. If the "other" range still exceeds the threshold, it will eventually be split again.
+
+In both of these examples, the split would only occur if the balance factor and split crossing heuristics determined the split would produce better results.
+
+## Why load-based splitting works
+
+CockroachDB creates a relatively even distribution of leaseholders throughout your cluster (leaseholders are a single replica of a range that both serve reads and coordinate writes operations; [learn more about leaseholders](architecture/replication-layer.html#leases)).
+
+However, without load-based splitting this distribution is unaware of the load present on any set of keys. This means that even with an equitable distribution of leases throughout the cluster, some leases will generate more traffic for the node that houses them than others. Because each node can only provide so much throughput, a single node can become a bottleneck for providing access to a subset of data in your cluster.
```

Comment:
> without load-based splitting this distribution is unaware of the load present on any set of keys

AIUI the cluster is what has the awareness, not the distribution.  #nitpicking

Suggest rewording to something like

> without load-based splitting, the cluster is unaware of the load present on any set of keys in this distribution

... assuming that a sentence like mine above does not introduce some subtle technical error, of course.